setup.py

```diff
@@ -43,8 +44,6 @@
         'plotly-express>=0.1.3',
         'pyarrow>=0.11.1',
         'pyyaml>=3.13',
-        'webviz_components>=0.0.1',
```

Comment:
At the same time as we loose this dependency, we could perhaps also remove the `dash <=` version constraint on https://github.com/equinor/webviz-config/blob/54b33ff8aee2b84555da81d3b6f309505fcf84cd/setup.py#L37.